serverless/python-template/GeneticStrategy.py

```diff
@@ -0,0 +1,81 @@
+import json
+from StrategyHandler import AbstractStrategyHandler
+import logging
+import deap
+from deap import base, creator, gp
+
+from GeneticProgram import GeneticProgram
+logger = logging.getLogger('TradingStrategy123')
+logger.setLevel(logging.INFO)
+
+
+class GeneticStrategy(AbstractStrategyHandler, GeneticProgram):
+    """"
+    A lambda doge baby.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        GeneticProgram.__init__(self)
+        self.individual = kwargs['individual']
+
+
+    def make_signal(self) -> int:
+        from StrategyHandler import BUY, SELL, IGNORE
+        doge = gp.compile(self.individual, self.pset)
+        outcome = doge([])
+        if outcome == self.buy:
+            self.signal = BUY
+        elif outcome == self.sell:
+            self.signal = SELL
+        elif outcome == self.ignore:
+            self.signal = IGNORE
+        else:
+            raise Exception("Unknown outcome produced by doge baby... singularity?!?")
+
+        return self.signal
+
+    @property
+    def used_indicators(self):
+        return ["RSI", "SMA"]  # TODO: add logic to auto-populate from individual
+
+    def rsi(self, input):
+        return self.get_indicator("RSI")
+
+    def sma50(self, input):
+        return self.get_indicator("SMA")
+
+    def ema50(self, input):
+        return self.get_indicator("RSI")
```

Comment:
@karla-b this looks like a mistake. can comment out until ema50 and other indicators are available